_lessons/2022-04-07-anticosti-historical-data-tutorial/anticosti-historical-data-tutorial.Rmd

```diff
@@ -11,11 +11,22 @@ output:
     self_contained: false
 ---
 
+This tutorial uses a tree damage dataset from Anticosti Island during a hemlock looper outbreak in 1973. The dataset can be found on the government of Canada's official [data portal](https://open.canada.ca/data/en/dataset/9dda09b0-649f-4002-b207-7b204eb81cbb). We would like to thank Anne Cotton-Gagnon, MSc, and Christian Hébert, PhD, from Natural Resources Canada and the Laurentian Forestry Centre for their insights on the original datasets and access to the reports and maps associated with such.
+
+The tutorial was built with the free and open-source software environment [R](https://www.r-project.org/) using the [leanr package](https://rstudio.github.io/learnr/) and and deployed using [Shiny](https://shiny.rstudio.com/). You can find the project folders and files in this [GitHub repository](https://github.com/Living-Data-Tutorials/Anticosti-Historical-Data).
+
+[Here is a direct link to our tutorial](https://vmglynn-ldp.shinyapps.io/LDT_Anticosti_Tutorial/#section-welcome-to-the-living-data-tutorials)
+
+The present tutorial was developed by Victoria Marie Glynn who is a PhD student at McGill University in Montréal, Québec, and Maxime Fraser Franco who is a PhD student at the Université du Québec à Montréal (UQAM) in Montréal, Québec. Please feel free to contact us if you have any question.
```

Comment:
I think it would be cleaner if we added affiliations in the YAML like this:
```
author:
  - name: Your Name
    affiliation: Your Institution
```

Maybe this works for email too? Like, adding a tag under `affiliation` as `url` followed by a mailto: link to your email.

Reply:
Sounds good, I will do this now @graciellehigino ! Thank you for the suggestion!